src/components/parameters/ParameterBaseString.svelte

```diff
@@ -17,27 +17,36 @@
   export let levelPadding: number = 20;
   export let parameterType: ParameterType = 'activity';
   export let use: ActionArray = [];
+  export let type: 'text' | 'password' = 'text';
 
   const dispatch = createEventDispatcher<{
     change: FormParameter;
     reset: FormParameter;
   }>();
 
   $: columns = `calc(${labelColumnWidth}px - ${level * levelPadding}px) auto`;
+
+  function handleChange(): void {
+    dispatch('change', formParameter);
+  }
+
+  const handleInput = (event: Event) => {
+    formParameter.value = (event?.target as HTMLInputElement).value;
```

Comment:
Does mutating the `value` field properly flow through the update cycle of the component? You can maybe test this by printing out `{formParameter.value}` outside of the input element as well and seeing if it updates as you type.